ferrocene/doc/internal-procedures/src/subtree-pulls.rst

```diff
@@ -78,7 +78,7 @@ Now run::
 The script will attempt to update all the registered subtrees in order
 until all succeed or the first one fails.
 
-You can also pass the name of a repository to only pull that specific repo::
+You can also pass the name of a repository to only pull that specific repo:
```

Comment:
This is intentional.

In rst,  the `::` at the end of a paragraph get turned into a single `:`, and the next paragraph becomes a code block.